README.md

```diff
@@ -135,6 +135,82 @@ systemd:
+```yaml
+variant: flatcar
+version: 1.0.0
+storage:
+  files:
+    - path: /opt/extensions/wasmcloud/wasmcloud-0.82.0-x86-64.raw
+      contents:
+        source: https://github.com/flatcar/sysext-bakery/releases/download/latest/wasmcloud-0.82.0-x86-64.raw
+    - path: /etc/sysupdate.d/noop.conf
+      contents:
+        source: https://github.com/flatcar/sysext-bakery/releases/download/latest/noop.conf
+    - path: /etc/sysupdate.wasmcloud.d/wasmcloud.conf
+      contents:
+        source: https://github.com/flatcar/sysext-bakery/releases/download/latest/wasmcloud.conf
+    - path: /etc/nats-server.conf
+      contents:
+        inline: |
+          jetstream {
+            domain: default
+          }
+          leafnodes {
+              remotes = [
+                  {
+                      url: "tls://connect.cosmonic.sh"
+                      credentials: "/etc/nats.creds"
+                  }
+              ]
+          }
+    - path: /etc/nats.creds
+      contents:
+        inline: |
+          <redacted>
+  links:
+    - target: /opt/extensions/wasmcloud/wasmcloud-0.82.0-x86-64.raw
+      path: /etc/extensions/wasmcloud.raw
+      hard: false
+systemd:
+  units:
+    - name: nats.service
+      enabled: true
+      dropins:
+        - name: 10-nats-env-override.conf
+          contents: |
+            [Service]
+            Environment=NATS_CONFIG=/etc/nats-server.conf
+    - name: wasmcloud.service
+      enabled: true
+      dropins:
+        - name: 10-wasmcloud-env-override.conf
+          contents: |
+            [Service]
+            Environment=WASMCLOUD_LATTICE=<redacted>
+    - name: systemd-sysupdate.timer
+      enabled: true
+    - name: systemd-sysupdate.service
+      dropins:
+        - name: wasmcloud.conf
+          contents: |
+            [Service]
+            ExecStartPre=/usr/lib/systemd/systemd-sysupdate -C wasmcloud update
+        - name: sysext.conf
+          contents: |
+            [Service]
+            ExecStartPost=systemctl restart systemd-sysext
+```
+
+This example uses Butane/Ignition configuration do the following customizations beyond simply including the image:
+
+1. Provide a different configuration to setup the nats-server to act as a leaf node to a pre-existing wasmCloud deployment (`/etc/nats-server.conf`).
+2. Provide a set of credentials for the nats-server leaf node to connect with (`/etc/nats.creds`).
+3. Override the bundled `NATS_CONFIG` environment variable to point it to the newly created configuration (`NATS_CONFIG=/etc/nats-server.conf`).
+4. Override the lattice the wasmCloud host is configured to connect (`WASMCLOUD_LATTICE=<redacted>`).
+
+This example add a configuration file for the bundled nats-server process to act as a leaf node for the wasmcloud host while making use of the credentials included in the Butane/Ignition configuration
```

Comment:
```suggestion
This example adds a configuration file for the bundled nats-server process to act as a leaf node for the wasmcloud host while making use of the credentials included in the Butane/Ignition configuration
```

Looks good otherwise, thanks.

Reply:
I actually didn't mean to include this paragraph, it was what I had originally started out with before I ended up with the bulleted list above, let me axe it. :)